ale_linters/ruby/ruby.vim

```diff
@@ -0,0 +1,46 @@
+" Author: Brandon Roehl - https://github.com/BrandonRoehl
+" Description: Ruby MRI for Ruby files
+
+function! ale_linters#ruby#ruby#Handle(buffer, lines) abort
+	" Matches patterns line the following:
+	"
+	" test.rb:3: warning: parentheses after method name is interpreted as an argument list, not a decomposed argument
+	" test.rb:8: syntax error, unexpected keyword_end, expecting end-of-input
+	let l:pattern = '\v^' . bufname(a:buffer) . ':(\d+): (warning: )?(.+)$'
```

Comment:
Does the filename need to be escsped here? Some characters could be interpreted as special regex characters.